src/js/media-container.js

```diff
@@ -19,58 +19,100 @@ template.innerHTML = `
     :host {
       box-sizing: border-box;
       position: relative;
-
-      /* Position controls at the bottom  */
-      display: inline-flex;
-      flex-direction: column-reverse;
+      display: inline-block;
 
       /* Max out at 100% width for smaller screens (< 720px) */
-      max-width: 100%;
+      // max-width: 100%;
```

Comment:
NOTE (opinionated): This should be applied externally, as it's a simple style adjustment, familiar to anyone who may want it, and would be more generically predictable styling/layout behavior for someone unfamiliar with media-chrome.

Reply:
I wish I had documented the issue better, but it's solving something... clearly something with smaller screens. An element limited to the width of its container doesn't feel super "unpredictable" to me. Is there more I'm missing?

Reply:
Hm. I disagree. That's not really how any html elements behave "out of box". With pretty much any visual HTMLElement, if you don't apply any CSS to them, the child will size itself and the parent will be sized by its children (with different default sizing/overflowing/etc. behavior based on the default display/layout rules of the elements in question).